packages/user-interface/src/pages/ThemeDetailsPage.tsx

```diff
@@ -34,6 +36,13 @@ const ThemeDetailsPage = ({
   children,
 }: Props) => {
   const intl = useIntl();
+  const { pushNotification } = useNotification();
+
+  useEffect(() => {
+    if (alert) {
```

Comment:
Is de alert prop nog wel nodig in deze pagina. Aangezien je het van buitenaf gewoon kan aanroepen met context?

Reply:
Alert prop is weg nu